dm_control/locomotion/walkers/assets/dog_v2/add_markers.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 The dm_control Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or  implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+"""Add markers to the dog model, used for motion capture tracking."""
+
+MARKERS_PER_BODY = {
+    "torso": (
+        (-0.25, 0, 0.13),
+        (-0.1, 0, 0.13),
+        (0.05, 0, 0.15),
+        (-0.25, -0.06, -0.01),
+        (-0.1, -0.08, -0.02),
+        (0.05, -0.09, -0.02),
+        (-0.25, 0.06, -0.01),
+        (-0.1, 0.08, -0.02),
+        (0.05, 0.09, -0.02),
+    ),
+    "C_6": ((0.05, -0.05, 0), (0.05, 0.05, 0)),
+    "upper_arm": ((0.04, 0, 0)),
+    "lower_arm": ((0.025, 0, 0), (0.025, 0, -0.1)),
+    "hand": ((0, -0.02, 0), (0, 0.02, 0), (0.02, -0.02, -0.08), (0.02, 0.02, -0.08)),
+    "finger": ((0.065, 0, 0)),
+    "pelvis": ((0.04, -0.07, 0.02), (0.04, 0.07, 0.02)),
+    "upper_leg": ((-0.03, 0.04, 0)),
+    "lower_leg": ((0.04, 0, 0), (-0.04, 0.035, 0)),
+    "foot": ((0, -0.02, 0), (0, 0.02, 0), (0, -0.02, -0.1), (0, 0.03, -0.1)),
+    "toe": ((0.055, 0, 0)),
+    "Ca_10": ((0, 0, 0.01)),
+    "skull": ((0.01, 0, 0.04), (0, -0.05, 0), (0, 0.05, 0), (-0.1, 0, 0.02)),
+}
+
+
+def add_markers(model):
+  """Add markers to the given model.
+
+  Args:
+      model: The model to add markers to.
+
+  Returns:
```

Comment:
If the function returns None you don't need to document it. Just drop it (and find any other places where such documentation is added).